coa-helm/templates/reverse-proxy.yaml

```diff
@@ -42,6 +42,9 @@ data:
         location /cromwell/ {
           proxy_pass http://{{ include "app.fullname" . }}-cromwell-svc:8000/;
         }
+        location /wds/ {
+          proxy_pass http://{{ include "app.fullname" . }}-wds-svc:8080/;
```

Comment:
Are these now the same in Azure and GCP? Or is it still worth having them separate?

Reply:
Perhaps not!

If we unify them: should that go in another PR? I think my answer would be "yes", but I don't feel strongly